### ISSUE TITLE
Data Issue: Tetracarpaea tasmannica

### ISSUE THREAD

**Issue by rossmounce** [wcornwell/taxonlookup#24]
It seems like _Tetracarpaea tasmannica_ gets left out because it's unresolved at TPL. APweb is fine with it though - it's not a poorly described, seldom-seen taxon: http://www.mobot.org/MOBOT/research/APweb/orders/saxifragalesweb.htm#Tetracarpaeaceae 

```
> plant_lookup_version_current()
[1] "1.1.1"
> lookup_table(c("Pinus ponderosa","Tetracarpaea tasmannica"),by_species=TRUE,missing_action="NA")
                               genus   family   order       group
Pinus ponderosa                Pinus Pinaceae Pinales Gymnosperms
Tetracarpaea tasmannica Tetracarpaea     <NA>    <NA>        <NA>
```

expected / desired output:

```
> lookup_table(c("Pinus ponderosa","Tetracarpaea tasmannica"),by_species=TRUE,missing_action="NA")
                               genus   family   order       group
Pinus ponderosa                Pinus Pinaceae Pinales Gymnosperms
Tetracarpaea tasmannica Tetracarpaea Tetracarpaeaceae Saxifragales Angiosperms
```

**Comment by wcornwell** [comment]
This is because TPL doesn't include Tetracarpaeaceae as a recognized family, and we're getting the species list family by family, so it's never ending up in the list.  Probably this is an error as Tetracarpaeaceae was in APG III https://en.wikipedia.org/wiki/APG_III_system

This is a new type of TPL error, though, so it's not an easy fix.  Will try to figure out how to add new families...

**Comment by wcornwell** [comment]
closed in favor of #37
